packages/storage/src/database/common/transactions.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  Filter,
+  FindOptions,
+  ReplaceOptions,
+  UpdateResult,
+  Document
+} from 'mongodb';
+import { zkDatabaseConstants } from '../../common/const.js';
+import { DB } from '../../helper/db-instance.js';
+import ModelBasic from '../base/basic.js';
+import ModelCollection from '../general/collection.js';
+
+export type TransactionType = 'deploy' | 'rollup';
+
+export type DbTransaction = {
+  transactionType: TransactionType;
+  databaseName: string;
+  tx: string;
+};
+
+export class ModelDbDeployTx extends ModelBasic<DbTransaction> {
+  private static INSTANCE: ModelDbDeployTx;
```

Comment:
I think `instance` is good, no need to UPPERCASE

Reply:
done